sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -337,12 +337,9 @@ namespace detail {
 namespace pi {
 
 // Report error and no return (keeps compiler from printing warnings).
-// TODO: Probably change that to throw a catchable exception,
-//       but for now it is useful to see every failure.
-//
 [[noreturn]] void die(const char *Message) {
```

Comment:
I think we should remove `[[noreturn]]` attribute.

Reply:
I removed it, and I have no warnings.